### PR TITLE
Category management

### PR DESCRIPTION
**Pull request by Cell5**
cat mgt front-end improvement

### CODE REVIEW

File: src/components/CategoryManagementComponents/MainCategoryManagement.vue [hunk 1]
```diff
@@ -8,28 +8,23 @@
         </b-row>
         <b-row>
             <b-col>
-                <b-container>
+                <b-container fluid>
                     <b-row>
                         <b-col cols="auto" class="list">
                             Hardware
+                            <img class="delete-img">
```
Comment (epsotto) [inline src/components/CategoryManagementComponents/MainCategoryManagement.vue]: Change this with fontawesome icons.

File: src/components/CategoryManagementComponents/MainCategoryManagement.vue [hunk 2]
```diff
@@ -10,21 +10,10 @@
...
-                        </b-col>
-                        <b-col cols="auto" class="list">
-                            Industrial Tools
-                            <img class="delete-img">
-                        </b-col>
-                        <b-col cols="auto" class="list">
-                            Auto Parts
+                        <b-col cols="auto" v-for="cat of cats" v-bind:key="cat.catName" class="list">
+                            <span>{{ cat.catName }}</span>
                             <img class="delete-img">
```
Comment (epsotto) [inline src/components/CategoryManagementComponents/MainCategoryManagement.vue]: Use fontawesome icons.

File: src/components/CategoryManagementComponents/MainCategoryManagement.vue [hunk 3]
```diff
@@ -10,21 +10,10 @@
...
-                            Electrical
-                            <img class="delete-img">
-                        </b-col>
-                        <b-col cols="auto" class="list">
-                            Industrial Tools
-                            <img class="delete-img">
-                        </b-col>
-                        <b-col cols="auto" class="list">
-                            Auto Parts
+                        <b-col cols="auto" v-for="cat of cats" v-bind:key="cat.catName" class="list">
```
Comment (epsotto) [inline src/components/CategoryManagementComponents/MainCategoryManagement.vue]: Class list is used but no declaration in css scripts.

File: src/components/CategoryManagementComponents/MainCategoryManagement.vue (hunk 1 above)
Comment (Cell5) [reply src/components/CategoryManagementComponents/MainCategoryManagement.vue]: done

File: src/components/CategoryManagementComponents/MainCategoryManagement.vue (hunk 2 above)
Comment (Cell5) [reply src/components/CategoryManagementComponents/MainCategoryManagement.vue]: done

File: src/components/CategoryManagementComponents/MainCategoryManagement.vue (hunk 3 above)
Comment (Cell5) [reply src/components/CategoryManagementComponents/MainCategoryManagement.vue]: please check, the css script declared in code at the bottom

**Pull request merged**